bitdefender/assets/logs/bitdefender.yaml

```diff
@@ -14,3 +49,656 @@ pipeline:
+          enabled: true
+          template: "%{params.events.computerIp}"
+          target: computerIp_extracted
+          replaceMissing: true
+        - type: grok-parser
+          name: Extract computer_ips
+          enabled: true
+          source: computerIp_extracted
+          samples:
+            - 10.10.10.10,10.10.10.11
+          grok:
+            supportRules: ""
+            matchRules: computer_ip_rule %{data:computer_ips:array(",")}
+        - type: attribute-remapper
+          name: Map `computer_ips` to `network.client.ip`
+          enabled: true
+          sources:
+            - computer_ips
+          sourceType: attribute
+          target: network.client.ip
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+    - type: pipeline
+      name: Processing of Antiexploit logs
+      enabled: true
+      filter:
+        query: service:antiexploit
+      processors:
+        - type: string-builder-processor
+          name: "%{params.events.computer_ip} - in attribute computer_ip_extracted"
+          enabled: true
+          template: "%{params.events.computer_ip}"
+          target: computer_ip_extracted
+          replaceMissing: true
+        - type: grok-parser
+          name: Extract computer_ips
+          enabled: true
+          source: computer_ip_extracted
+          samples:
+            - 10.10.10.10,10.10.10.11
+          grok:
+            supportRules: ""
+            matchRules: computer_ip_rule %{data:computer_ips:array(",")}
+        - type: attribute-remapper
+          name: Map `computer_ips` to `network.client.ip`
+          enabled: true
+          sources:
+            - computer_ips
+          sourceType: attribute
+          target: network.client.ip
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - type: string-builder-processor
+          name: "%{params.events.detection_username} - in attribute
+            detection_username_extracted"
+          enabled: true
+          template: "%{params.events.detection_username}"
+          target: detection_username_extracted
+          replaceMissing: true
+        - type: grok-parser
+          name: Extract detection_usernames
+          enabled: true
+          source: detection_username_extracted
+          samples:
+            - john,alice
+          grok:
+            supportRules: ""
+            matchRules: detection_username_rule %{data:detection_usernames:array(",")}
+        - type: attribute-remapper
+          name: Map `detection_usernames` to `usr.name`
+          enabled: true
+          sources:
+            - detection_usernames
+          sourceType: attribute
+          target: usr.name
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+    - type: pipeline
+      name: Processing of Netwrok Attack Defense logs
```

Comment:
Can't make a suggestion for some reason, but "Netwrok" should be "Network" here

Reply:
Done